src/Kentico.Xperience.Algolia/Indexing/DefaultAlgoliaClient.cs

```diff
@@ -0,0 +1,224 @@
+using Algolia.Search.Clients;
+using Algolia.Search.Models.Common;
+using CMS.ContentEngine;
+using CMS.Core;
+using CMS.DataEngine;
+using CMS.Helpers;
+using CMS.Helpers.Caching.Abstractions;
+using CMS.Websites;
+using Kentico.Xperience.Algolia.Services;
+using Newtonsoft.Json.Linq;
+
+namespace Kentico.Xperience.Algolia.Indexing;
+
+/// <summary>
+/// Default implementation of <see cref="IAlgoliaClient"/>.
+/// </summary>
+internal class DefaultAlgoliaClient : IAlgoliaClient
+{
+    private readonly IAlgoliaIndexService algoliaIndexService;
+    private readonly IInfoProvider<ContentLanguageInfo> languageProvider;
+    private readonly IInfoProvider<ChannelInfo> channelProvider;
+    private readonly IConversionService conversionService;
+    private readonly IContentQueryExecutor executor;
+    private readonly IProgressiveCache cache;
+    private readonly ISearchClient searchClient;
+
+    internal const string CACHEKEY_STATISTICS = "Algolia|ListIndices";
```

Comment:
This isn't needed anymore, right?

Reply:
I thought I have already removed this. Pushed removal.